script/Ciel/Day 9/09_01_CIEL09_3_1.txt

```diff
@@ -1,12 +1,12 @@
 [sha:5c739aa6b10dc78abb740cc9ad4c4fd7d6ee3733]{
 -- Page 10009, Offset 22962.
 -- 「なに？　やっぱり、シエルさんと学生ごっこやってる志貴クンには荷が重い？」
-"What? Is that too much for someone like you, pretending to be a student like Ciel-san?"
+"What? Is it too much for someone who's playing make-believe student like Ciel-san, after all?"
```

Comment:
"like Ciel-san" sounds like Ciel is the one she's talking about, when in fact she's just comparing the two. My take is a little bit modified from the source but makes more sense in EN
> What? Too much to bear for you? Still playing make-believe student like Ciel-san?